src/VisualStudio/IntegrationTest/TestUtilities/InProcess/SolutionExplorer_InProc.cs

```diff
@@ -379,6 +384,10 @@ public void CleanUpOpenSolution()
 
             if (dte.Debugger.CurrentMode != EnvDTE.dbgDebugMode.dbgDesignMode)
             {
+                // Close the Find Source window in case it's open.
```

Comment:
Is it too much work to add a "is there is dialog open" function?

Reply:
I don't trust the dialog system enough to think that they could all be closed in the same way. Normally we would use UI Automation to find the dialog by ID, but in this case I wasn't sure what the ID is and the current behavior is causing numerous failures in the master branch.

Reply:
I'm leaving the issue open and adding a comment there that this workaround should be removed or _substantially_ improved.